synapse/lib/msgpack.py

```diff
@@ -40,7 +42,8 @@ def en(item):
         return pakr.pack(item)
     except Exception:
         pakr.reset()
-        raise
+        mesg = f'Cannot serialize: {item!r}'
+        raise s_exc.NotMsgpackSafe(mesg=mesg)
```

Comment:
we should also grab object class name here.

Reply:
discussed